packages/teams-js/test/public/dialog.spec.ts

```diff
@@ -452,12 +458,38 @@ describe('Dialog', () => {
           framedMock.setRuntimeConfig({ apiVersion: 1, supports: {} });
           expect.assertions(4);
           try {
-            dialog.submit('someResult', ['someAppId', 'someOtherAppId']);
+            return await dialog.submit('someResult', ['someAppId', 'someOtherAppId']);
```

Comment:
Can't we just call `dialog.submit`?

Reply:
we don't, they are redundant, it was an oversight when I changed the code